### DISCUSSION TITLE
Playlist Remote URL

### DISCUSSION THREAD

**Discussion by radiobijoux** [AzuraCast/AzuraCast#4350] [Q&A]
I've discovered how (I think) to add a remote relay stream to run, but thus far the playlist doesn't function as required.  I added a playlist and selected "Remote URL Playlist" as the source.  I inserted the appropriate URL (and tested directly in a browser to make certain that it works), and added the scheduled times.  Since this is another station, the stream is always active so no need to worry about timings, etc.  It should just connect at the appropriate time and begin playing their stream over mine.  The remote stream is never picked up however.  I've tested it twice and then again when the scheduled day/time arrived, but my station simply continued to play.  I set a buffer of 10 seconds, then 5, etc. and also checked the "interrupt playing song" in the advanced tab as well just to make sure, but still nothing.  There is nothing in the Documents that I can find about what the remote URL playlist, although I may have just missed that.  Any thoughts or ideas?  Also, how long a lead time should I give a playlist edit or addition to become active?

**Comment by GeertSurkijn** [comment]
I am struggling with the same, today updated to the latest stable release. I was wondering how the remote stream will see Azuracast? Can it be that the remote station is not allowing to do this? 

In my case, the remote is our radio studio, having an Icecast stream. Not published, but open if you know the URL. I can listen with any player, but in Azuracast nothing seems to start, get a little silence and then the playlists start streaming randomly.
Also, I am not sure what option I should enable in the advanced settings, nor it is not clear if we need to configure something special in the Liquidsoap settings. I noticed this in the config:

```
playlist_gothville_studio_link = mksafe(input.http(max=15., "http://xxxxxxxxxx:8000/stream"))
playlist_gothville_studio_link = audio_to_stereo(id="stereo_playlist_gothville_studio_link", playlist_gothville_studio_link)
playlist_gothville_studio_link = cue_cut(id="cue_playlist_gothville_studio_link", playlist_gothville_studio_link)
ignore(playlist_gothville_studio_link)

```

But even if I add a schedule, it is not being set in the radio = switch() setting of liquidsoap where you normaly notice the playlist schedules....

<img width="419" alt="2021-06-28 18_10_06-Playlists - Wavebox" src="https://user-images.githubusercontent.com/8115472/123669287-3728f300-d83c-11eb-9c61-a72719e1728f.png">
<img width="420" alt="2021-06-28 18_09_25-Playlists - Wavebox" src="https://user-images.githubusercontent.com/8115472/123669289-37c18980-d83c-11eb-8215-2585ba4cdd36.png">
<img width="422" alt="2021-06-28 18_10_28-Playlists - Wavebox" src="https://user-images.githubusercontent.com/8115472/123669291-385a2000-d83c-11eb-9def-98d798d5e76f.png">

**Comment by GeertSurkijn** [comment]
> I've discovered how (I think) to add a remote relay stream to run, but thus far the playlist doesn't function as required. I added a playlist and selected "Remote URL Playlist" as the source. I inserted the appropriate URL (and tested directly in a browser to make certain that it works), and added the scheduled times. Since this is another station, the stream is always active so no need to worry about timings, etc. It should just connect at the appropriate time and begin playing their stream over mine. The remote stream is never picked up however. I've tested it twice and then again when the scheduled day/time arrived, but my station simply continued to play. I set a buffer of 10 seconds, then 5, etc. and also checked the "interrupt playing song" in the advanced tab as well just to make sure, but still nothing. There is nothing in the Documents that I can find about what the remote URL playlist, although I may have just missed that. Any thoughts or ideas? Also, how long a lead time should I give a playlist edit or addition to become active?

Did you find some kind of solution for it?

**Reply by radiobijoux** [reply]
At this point, no.  I'm waiting to find out if the rest of the calendaring/scheduling portion is going to work for my purposes.  Libretime has an awesome calendaring and scheduling system, but they have a lot of other issues to work out before their solution becomes solid.  I'll probably have to continue with some type of commercial system until both or one of these options becomes viable for my use.

**Comment by thanhnc-iot** [comment]
I am stuck on the same, it take me a day. Can it do remote URL playlist?

**Comment by n1ce-0ne** [comment]
same, doesn't seem to work

my goal:

i have a local playlist playing 24/7 with autodj.

my friends on another site have ac too,

if i stream on my friends site i'd like to relay my friends ac stream through my ac
so any friends on my site can enjoy too

**Reply by n1ce-0ne** [reply]
sorted this, if anyone is having issues adding a remote stream:
test with another stream, my main issue was the stream, i tried another & boom it worked

to fix it i had to re-stream the source with a php file then load the php file into ac,
you loose titles but at least i get the music [which is me live streaming anyway]
